lib/hopper/configuration.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'active_support/time'
+
+class Hopper::Configuration
+  class << self
+    attr_accessor :configuration
+
+    DEFAULTS = {
+      publish_retry_wait: 1.minute
+    }.freeze
+
+    def load(configuration)
+      @configuration = DEFAULTS.merge(configuration)
+    end
+
+    def method_missing(method)
+      method = method.to_s.sub(/=$/, "").to_sym
```

Comment:
It looks like you're deliberately turning writes into reads here: if I were to do `described_class.publish_retry_wait = 1.minute`, this would instead return the existing value of `described_class.publish_retry_wait` instead of setting it to the expected value; is this what you intended?

Reply:
If you want to disable writes, you could just not respond to methods ending with `=`

Reply:
You are correct, copied the code without paying too much attention, I've removed it now